tests/test_tools.py

```diff
@@ -160,3 +165,65 @@ def test_get_package_info(monkeypatch, inputs, expected):
     )
     actual_metadata = get_package_info(inputs[0], metadata=inputs[1])
     assert actual_metadata == expected
+
+
+@pytest.mark.parametrize(
+    "inputs, expected_mu",
+    [
+        # Test whether the function returns the correct mu
+        (  # C1: No density or packing fraction (only for known material), expect to get mu from database
```

Comment:
"C1 composition and wavelength provided, known material. Expect to get mu for fully dense material"